client/src/containers/explore-map/component.tsx

```diff
@@ -24,6 +24,16 @@ const ExploreMap = () => {
       <Media greaterThanOrEqual="sm">
         <Sidebar />
       </Media>
+      <Media greaterThanOrEqual="sm">
```

Comment:
You don't need to use `<Media>` when you can use a css/tailwind breakpoint. This is a common error we ended up. Media should only be used for layout changes
```
sm:block hidden
```